lab_1/main.py

```diff
@@ -49,7 +76,16 @@ def get_top_n_words(freq_dict: dict, top_n: int) -> list:
     top_n = 1
     --> ['happy']
     """
-    pass
+    top_words = []
+    try:
```

Comment:
`try ... except` blocks are usually wrap the line where the Exception is likely to be raised.

Take a look at `isinstance` for type-checking.